code-challenges.test.js

```diff
@@ -21,9 +29,41 @@ const colors1 = ["purple", "blue", "green", "yellow", "pink"]
 const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
+describe("dropFirst", () => {
+    it("takes in an array, removes the first item from the array and shuffles the remaining content.", () => {
+    const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+    const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+    expect(dropFirst(colors1)).toEqual(expect.arrayContaining(["yellow", "blue", "pink", "green"]))
+    expect(dropFirst(colors2)).toEqual(expect.arrayContaining(["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]))
+    })
+    })
+
+    // ReferenceError: dropFirst is not defined
 
 // b) Create the function that makes the test pass.
 
+// Pseudocode:
+// Create a function called dropFirst that takes in an array, removes the first item from the array and shuffles the remaining content
+// Taking in one parameter, array
+// function will first use the .shift method to remove the first value
+// function will then use the math.random method to randomize the returning array
+// .length to iterate over the array
+// return the new array
+
+const dropFirst = (array) => {
+      array.shift()
+      array.sort(Math.floor(Math.random() * array.length))
+      return array
+  }
+
+// Getting two error messages 
+  // ReferenceError: Cannot access 'dropFirst' before initialization
+  // ReferenceError: math is not defined
+  // refactored code, then got new error
+  // TypeError: The comparison function must be either a function or undefined
```

Comment:
This is a very helpful error as it is hinting that sort needs a function to operate in that manner.  In other words, sort needs to become a higher-order function by passing it an anonymous function to allow it to randomize the order.